### PR TITLE
font size changing

### PR DESCRIPTION
**Pull request by allen0346**
function completed with simple animation.

### CODE REVIEW

File: src/components/FontChangeButton.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Styles from './FontChangeButton.scss'
```
Comment (nickhsine) [inline src/components/FontChangeButton.js]: use ```styles```.

File: src/components/FontChangeButton.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class FontChangeButton extends React.Component {
+  constructor() {
+    super()
+    this.state={ active: true }
+    this.anima = this.anima.bind(this)
+  }
+  anima() {
+    this.state.active === true ? this.setState({ active:false }) : this.setState({ active:true })
```
Comment (nickhsine) [inline src/components/FontChangeButton.js]:
```
this.setState({
  active: !this.state.active
})
```

**Comment by garfieldduck** [comment]
👍 👍 👍

File: src/components/FontChangeButton.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }
+  getFontSize(fontSize) {
```
Comment (nickhsine) [inline src/components/FontChangeButton.js]: rename it as ```changeFontSize``` or ```setFontSize``` ?

File: src/components/article/ShareBt.js [hunk 4]
```diff
@@ -4,20 +4,37 @@ import classNames from 'classnames'
...
+        </TwitterButton>
+        <a href={lineUrl} className={classNames(styles.bt, styles.line)}>
+          <img src={lineIcon} />
+        </a>
+      </div>
+    )
+  }
 }
+
+export { ShareBt }
```
Comment (nickhsine) [inline src/components/article/ShareBt.js]: Revert this file.

**Pull request merged**